src/main/kotlin/no/skatteetaten/aurora/boober/utils/StringUtils.kt

```diff
@@ -14,4 +16,6 @@ fun String.ensureStartWith(startWith: String, seperator: String = ""): String {
     }
     return "$startWith$seperator$this"
 
-}
\ No newline at end of file
+}
+
+fun String.removeExtension(): String = FilenameUtils.removeExtension(this)
```

Comment:
Rent semantisk tror jeg at jeg hadde foretrukket å ha denne i en egen FilenameUtils-fil. Dette er tross alt ikke en generell String-manipulasjonsfunksjon selv om den opererer på Strings. Kanskje vi kunne deklarert et Filename-alias for String i denne filen også? Da kan vi sende rundt Filename-objekter istedenfor Strings.

Reply:
Sure